index.js

```diff
@@ -19,9 +19,31 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    await Recipe.create({
+      title: "Hot Dog Fingers",
+      level: "UltraPro Chef",
+      ingredients: ["5 hot dogs", "5 human fingers", "100g of cut off nails"],
+      cuisine: "Anthropofagic",
+      dishType: "snack",
+      duration: 120,
+      creator: "Chef Hannibal",
+    });
+    console.log("Hot Dog Fingers");
   } catch (error) {
     console.log(error);
   }
+
+  await Recipe.insertMany(data);
+  data.forEach((Recipe) => console.log(Recipe.title));
+
+  await Recipe.findOneAndUpdate(
+    { title: "Rigatoni alla Genovese" },
+    { duration: 100 }
+  );
+  console.log("Great Success!");
+
+  await Recipe.deleteOne({ title: "Carrot Cake" });
```

Comment:
In the last iteration they ask you to close the connection with mongoDB. You're missing that step here at the end:

```
await mongoose.disconnect();
```